src/searchbar/__tests__/__snapshots__/SearchBar.js.snap

```diff
@@ -9,7 +9,7 @@ exports[`SearchBar wrapper component should render a default SearchBar if wrong
   onChangeText={[Function]}
   onClear={[Function]}
   placeholderTextColor="#86939e"
-  platform="wrong-platform"
```

Comment:
The goal of this test is to check that the default searchbar is rendered when a, unknown platform is specified (fallback pattern). Could you let `wrong-platform` or whatever different than `ios`, `android` or `default` ?

Reply:
so should i change wrong-platform inside of default ?

Reply:
Yes, anything else that `ios`, `android` or `default`. `wrong-platform` was good.

Reply:
okay dear